### DISCUSSION TITLE
Support mobile/smartphone platforms (Android, iOS, ...)

### DISCUSSION THREAD

**Discussion by dumblob** [freenet/freenet-core#811] [General]
Any plans to support "out of the box" (i.e. through F-Droid or Apple app repositories) mobile platforms?

I think that would be a true breakthrough as these are esentially by far the most widespread communication tools people use nowadays.

**Comment by sanity** [comment]
Yes, we'll support mobile.

**Reply by defiling9046** [reply]
Any updates regarding Freenet on Android?
